.github/workflows/npm-publish.yml

```diff
@@ -16,7 +16,10 @@ jobs:
         with:
           node-version: 20
       - run: npm ci
-      - run: npm test
+      - run: |
```

Comment:
Why don't we use the wait action as in the test workflow?

```
      - uses: ifaxity/wait-on-action@v1.1.0
        with:
          delay: 1
          timeout: 30000
          resource: tcp:localhost:8082
```

Reply:
Good suggestion. I'm going to change that